src/components/Card/Card.js

```diff
@@ -132,19 +136,6 @@ const styles = StyleSheet.create({
     width
   }),
   imageHeader: {height: '100%', width: '100%', borderRadius: 45},
-  constainerHeader: {
-    flexDirection: 'row',
-    padding: 8
-  },
-  contentHeader: {
-    borderRadius: 45,
-    borderWidth: 0.2,
-    borderColor: COLORS.black50,
-    width: 36,
-    height: 36,
-    justifyContent: 'center',
-    alignItems: 'center'
-  },
   container: (height) => ({
     borderWidth: 0.5,
     borderRadius: 8,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The styles `containerHeader` and `contentHeader` have been removed. If these styles were being used elsewhere in the code, this could potentially break those components. Please ensure that these styles are not being used elsewhere before removing them.



<!-- This is an auto-generated comment by OSS CodeRabbit -->